### DISCUSSION TITLE
Command which will open SSB2 menu for player

### DISCUSSION THREAD

**Discussion by Envqu** [BG-Software-LLC/SuperiorSkyblock2#2199] [Pending Feature Requests]
Please add command - something like /is admin opengui [guiName] [player] - for example: /is admin opengui counts envqu - opens counts gui from SSB2 for player named envqu

**Comment by OmerBenGera** [comment]
Just make the player to execute /is counts

**Comment by Envqu** [comment]
Or add permission to block possibility to check other islands counts
